cli/commands/dashboard.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 const (
-	dashboardContainerName string = "outrigger-dashboard"
-	dashboardImageName     string = "outrigger/dashboard:latest"
```

Comment:
I made these lower camel case to keep them as private constants. Do we have a use case to export them?

Reply:
Did not mean to make them public (by convention), was just looking at some approaches. i can revert that.